bundle/bundle.go

```diff
@@ -0,0 +1,11 @@
+package bundle
+
+import (
+	"embed"
+)
+
+// Note: this will bundle the tests as well, but since that has negligible impact on the size of the binary,
+// it's preferable to filter them out from the bundle than to e.g. create a separate directory for tests
+//
+//go:embed *
```

Comment:
💭 Probably means that _this_ file makes it into the bundle, too. But it should be ignored...?

Reply:
https://pkg.go.dev/embed#hdr-Directives

> If a pattern names a directory, all files in the subtree rooted at that directory are embedded (recursively), except that files with names beginning with ‘.’ or ‘_’ are excluded. 

Does that mean the file could be named `_bundle.go`, or would that not work with Go? 🤔

Reply:
I was hoping that `*` explicitly included files starting with `.` -- we need it for `.manifest`.

Reply:
Oh right, you'll need to use `all` for that, I think

Reply:
✔️ Double-checked: `.manifest` is there, and `bundle.go` is, too.

Reply:
Yay!